ntbs-service/Pages/ClinicalDetails/Edit.cshtml

```diff
@@ -0,0 +1,412 @@
+@page "{handler?}"
+@model ntbs_service.Pages_ClinicalDetails.EditModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
+@using ntbs_service.Models
+
+@{
+    ViewData["Title"] = "Notification - Clinical Timeline";
+}
+@{string errorClassName = "nhsuk-form-group--error";}
+<div><a asp-page="/Patients/Index">Back to List</a></div>
+
+<h1>Clinical Details</h1>
+<!-- TODO: Split this into multiple partial pages for readability -->

```

Comment:
This file has gotten very large - @jasiekmiko  do you think we should split this into several partials? I.e. keep all the sites together, and the clinical dates, etc.

Should I leave it for now though and prioritise other tickets?